modules/project_cleanup/function_source/main.go

```diff
@@ -247,6 +328,49 @@ func invoke(ctx context.Context) {
 		}
 	}
 
+	tagKeyAgeFilter := func(tagKey *cloudresourcemanager3.TagKey) bool {
+		tagKeyCreatedAt, err := time.Parse(time.RFC3339, tagKey.CreateTime)
+		if err != nil {
+			logger.Printf("Fail to parse CreateTime for tagKey [%s], skip it. Error [%s]", tagKey.Name, err.Error())
```

Comment:
suggestion: `skip` > `skipping`

Reply:
@apeabody we have another 2 cases of previously existing messages with "skip it".
Should we update them all for consistency?

Reply:
fixed